wiki/osu!_originals/en.md

```diff
@@ -100,6 +100,18 @@ These songs are part of the [osu! James Landino Collection EP](https://fanlink.t
 | --: | :-: | :-- |
 | [![SoundCloud](/wiki/shared/link/SoundCloud.png)](https://soundcloud.com/dksslqj/gehenna "SoundCloud") | [![FA](/wiki/shared/link/FA.png)](https://osu.ppy.sh/beatmaps/artists/tracks?artist=Rabbit%20House&query=%22Gehenna%22 "Featured Artist listing") | Rabbit House - Gehenna |
 
+#### [tomspicy](https://osu.ppy.sh/beatmaps/artists/437)
+
+*For the announcement news post, see: [New Featured Artist: tomspicy](https://osu.ppy.sh/home/news/2024-09-14-new-featured-artist-tomspicy)*
+
+These songs are part of the [tomspicy osu! collection 01 EP](https://osu.ppy.sh/beatmaps/artists/437#album-2135)
```

Comment:
Maybe add a colon or a period at the end?
```suggestion
These songs are part of the [tomspicy osu! collection 01 EP](https://osu.ppy.sh/beatmaps/artists/437#album-2135):
```
```suggestion
These songs are part of the [tomspicy osu! collection 01 EP](https://osu.ppy.sh/beatmaps/artists/437#album-2135).
```

There's a similar sentence in the James Landino section.

Reply:
Prooooooobably fine as is, esp. since it matches the james landino case as it stands.